docker-compose.dev.yml

```diff
@@ -21,23 +21,52 @@ services:
     volumes: *sbt-volumes
     stdin_open: true # docker run -i
     tty: true        # docker run -t
+    ports:
+      - 9000:9000
+      - 8457:8457
+    command: -jvm-debug "*:8457" ~run -Dconfig.file=conf/application.dev.conf
 
   civiform-shell:
-    container_name: civiform-shell
     image: civiform-dev
     entrypoint: /bin/bash
     profiles: ["shell"]
     volumes: *sbt-volumes
     stdin_open: true # docker run -i
     tty: true        # docker run -t
 
+  localstack:
+    ports:
```

Comment:
nit: can this be expose instead of the port mapping

Reply:
Unfortunately, no.  We use the ports externally in our startup scripts to poll for the emulators being up.